pkg/querier/metadata_handler_test.go

```diff
@@ -158,30 +158,6 @@ func TestMetadataHandler_Success(t *testing.T) {
 				}
 			`,
 		},
-		"metric": {
```

Comment:
was this test removed on purpose?

Reply:
Yes, I removed the part that enforced the metric parameter from there since it's enforced earlier in the ingesters.

Reply:
Right. We expect the metric to be filtered by ingesters, so we're not filtering it anymore in the querier. Makes sense to me.